packages/components/src/Form/Inputs/InputFilters/InputFilters.tsx

```diff
@@ -0,0 +1,140 @@
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+
+import styled from 'styled-components'
+import React, { FC, useState, useRef, MouseEvent } from 'react'
+import { Select } from '../Select'
+import { InputText } from '../InputText'
+import { Icon } from '../../../Icon'
+import { IconButton } from '../../../Button'
+import { Chip } from '../../../Chip'
+import { Space } from '../../../Layout'
+
+export interface AvailableFilter {
+  value: string
+}
+
+export interface InputFiltersProps {
+  className?: string
+  availableFilters: AvailableFilter[]
+  fieldFilters?: string[]
+  hideFilterIcon?: boolean
+}
+
+const InputFiltersLayout: FC<InputFiltersProps> = ({
+  availableFilters,
+  className,
+  fieldFilters,
+  hideFilterIcon = false,
+}) => {
+  const [values, setValues] = useState(fieldFilters || [])
+  const [filterLookupName, setFilterLookupName] = useState('')
+
+  const inputRef = useRef<null | HTMLInputElement>(null)
+  const isClearable = values.length > 0
+
+  const clearFilters = (event: MouseEvent<HTMLSpanElement>) => {
+    event.stopPropagation()
+    setValues([])
+  }
+
+  function focusInput() {
+    inputRef.current && inputRef.current.focus()
+  }
+
+  function handleFilterLookupChange(newValues: any) {
+    setFilterLookupName('')
+    setValues([...values, newValues])
+  }
+
+  const editFilter = (event: MouseEvent<HTMLSpanElement>) => {
+    // alert("You can't do that. Yet!")
+    console.log('event: ', event)
+    event.stopPropagation()
+  }
+
+  return (
+    <div className={className} onClick={focusInput}>
+      {!hideFilterIcon && (
+        <Icon mx="xsmall" name="Filter" size={20} color="ui4" />
+      )}
+      <Space gap="xsmall">
+        {values.map((value, i) => (
+          <Chip key={i} onClick={editFilter}>
+            {value}
+          </Chip>
+        ))}
+
+        <Select
+          autoResize
+          indicator={false}
+          isFilterable
+          onChange={handleFilterLookupChange}
+          options={availableFilters}
+          placeholder="Filter List"
+          ref={inputRef}
+          value={filterLookupName}
+        />
+      </Space>
+      {isClearable && (
+        <IconButton icon="Close" label="Clear filters" onClick={clearFilters} />
+      )}
+    </div>
+  )
+}
+// padding-left: ${props.theme.space.xsmall}
+export const InputFilters = styled(InputFiltersLayout)`
+  align-items: center;
+  border: solid 1px ${({ theme }) => theme.colors.ui2};
+  border-radius: ${({ theme: { radii } }) => radii.medium};
+  display: flex;
+  flex: 2;
+
+  ${Space} {
+    ${(props) =>
+      props.hideFilterIcon && `padding-left: ${props.theme.space.xxsmall}`}
+  }
```

Comment:
Recommend taking a slightly simpler approach here - always applying padding to the `Input` itself and then just adjusting the padding on `Icon` to compensate for pre-existing padding on the left.

```suggestion
  padding: 0 ${({ theme }) => theme.space.xxsmall};

  ${Icon} {
    margin-left: ${({ theme }) => theme.space.xxsmall};
    margin-right: ${({ theme }) => theme.space.xsmall};
  }
```